packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -395,8 +386,7 @@ export default class SpecPreview extends GlimmerComponent<Signature> {
   @service private declare realmServer: RealmServerService;
   @service private declare cardService: CardService;
   @service private declare playgroundPanelService: PlaygroundPanelService;
-  @tracked private _selectedCardId?: string;
-  @tracked private newCardJSON: LooseSingleCardDocument | undefined;
+  @tracked private _selectedCard?: Spec;
```

Comment:
This was one of the saving grace. Using a getter that included lines like this always caused a re-render. By assinging at the unit of the card, we were able to keep th edit of the card stable

```
this.cards.find((card) => card.id === card.id);
```